package/scripts/install_prereqs_for_wheels.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+install_prereqs_for_mac() {
+  HOMEBREW_NO_AUTO_UPDATE=1
+  HOMEBREW_NO_INSTALL_CLEANUP=1
+  brew list openssl@3 &> /dev/null || brew install openssl@3
+  brew list libcsv &> /dev/null && brew uninstall libcsv
+  # Use the uuid from framework
+  brew list ossp-uuid &> /dev/null && brew uninstall ossp-uuid
+  brew list cmake &>/dev/null || brew install cmake
+  brew list automake &> /dev/null || brew install automake
+  brew list pkg-config &> /dev/null || brew install pkg-config
+}
+
+install_prereqs_for_centos7() {
+  echo "Installing minimal dependencies..."
+  yum install -y centos-release-scl && yum install -y devtoolset-11 &&
+    yum install -y -q deltarpm &&
+    yum update -y -q &&
+    yum install -y -q epel-release &&
+    yum install -y -q which wget git &&
+    yum install -y -q autoconf automake libtool unzip &&
+    yum install -y -q cmake3 patch &&
+    yum install -y -q perl perl-IPC-Cmd &&
+    yum install -y -q libuuid libuuid-devel &&
+    yum install -y -q curl libcurl-devel &&
+    echo "Installing minimal dependencies DONE"
+}
+
+install_genomicsdb_for_mac() {
+    PREFIX_DIR=${PREFIX_DIR:-$(mktemp -d)}
+    GENOMICSDB_DIR=$PREFIX_DIR/GenomicsDB
+    GENOMICSDB_HOME=/usr/local
+    export $GENOMICSDB_HOME
+    export MACOSX_DEPLOYMENT_TARGET=12.1
+    export OPENSSL_ROOT_DIR=$(brew --prefix openssl@3)
+    rm -fr $GENOMICSDB_DIR
+    git clone https://github.com/GenomicsDB/GenomicsDB.git -b develop $GENOMICSDB_DIR
+    pushd $GENOMICSDB_DIR
+    echo "Starting GenomicsDB build"
+    mkdir build && cd build &&
+    cmake -DCMAKE_INSTALL_PREFIX=$GENOMICSDB_HOME -DCMAKE_PREFIX_PATH=$OPENSSL_ROOT_DIR -DPROTOBUF_ROOT_DIR=./protobuf-install -DAWSSDK_ROOT_DIR=./aws-install -DGCSSDK_ROOT_DIR=./gcs-install -DBUILD_EXAMPLES=False -DDISABLE_MPI=True -DDISABLE_OPENMP=True -DUSE_HDFS=False .. &&
+        make -j4 && make install
+    popd
+    if [[ -f $GENOMICSDB_HOME/lib/libtiledbgenomicsdb.dylib ]]; then
+        echo "Building GenomicsDB in $GENOMICSDB_DIR DONE"
+    else
+        echo "Something wrong with building GenomicsDB at $GENOMICSDB_HOME"
+        exit 1
+    fi
+}
+
+install_genomicsdb_for_centos7() {
+    source /opt/rh/devtoolset-11/enable
+    INSTALL_PREFIX=/usr/local
+    export OPENSSL_ROOT_DIR=$INSTALL_PREFIX
+    export LD_LIBRARY_PATH=$INSTALL_PREFIX/lib64:$INSTALL_PREFIX/lib:$LD_LIBRARY_PATH
+    git clone https://github.com/GenomicsDB/GenomicsDB.git -b develop GenomicsDB-build
```

Comment:
Shouldn't we be using `master` branch here?

Reply:
Yes, we should be using `master`, but have not synched up with `develop` yet.